### ISSUE TITLE
When adapt frame work is changed to bigger screen resolution bg doesn't fill

### ISSUE THREAD

**Issue by mike-st** [cgkineo/adapt-background-switcher#6]
Hi I recently downloaded this version for adapt 2.0 and works great except for my adapt framework has been customized to have the largest screen resolution at 1300px instead of the 900px default in adapt-contrib-vanilla/theme.json. The background switcher isn't adapting to this customization so there is a bar of the background color instead of background image showing on the right hand side. That said I believe this can be fixed using css/less percentage values instead of pixel values

**Comment by LukaszGrela** [comment]
Hi Mike,

in less you have 'cover' defined and sizes are in percentage
```
            .background-switcher-background {

                background-size: cover;
                background-position: 50% 0;
                position: fixed;
                z-index: 0;
                width: 100%;
                height: 100%;
                top: (@navigation-padding * 2) + @icon-size;                                     
                opacity: 0;

                transition: opacity 1s;
            }
```

**Comment by LukaszGrela** [comment]
@mike-st can you update about your issue? did you managed to resolve it?

**Comment by mike-st** [comment]
Hi Lukasz
Apologies for not getting back right away. I have fixed it by changing one line off css in the less file. I added in max-width: 100% !important; on the class ".page.background-switcher-active .background-switcher-container .background-switcher-background". Here are some pics show the issue and the fix. 

![background-switcher-issue](https://cloud.githubusercontent.com/assets/10455453/10137253/8ca46a24-65c6-11e5-8ac2-2f4456aaa405.jpg)
![fixed-background-switcher-issue](https://cloud.githubusercontent.com/assets/10455453/10137254/8ca46600-65c6-11e5-99c0-3c182961430f.jpg)
